uwblueprint/ccbc
# Admin dashboard redesign

## Notion ticket link

[Admin Dashboard Redesign](https://www.notion.so/uwblueprintexecs/Admin-Dashboard-Redesign-cbe3fac4aceb4d4f88921f31fad2ed05)



## Implementation description
* Change Published column to Status column with `Published` and `Draft` tags
* Add Edit review, Preview, Delete icons and tooltips



## Steps to test
1. Verify both published and draft tags display
2. Verify all three icons display for every row with the correct tooltips



## What should reviewers focus on?
* Code correctness


## Checklist
